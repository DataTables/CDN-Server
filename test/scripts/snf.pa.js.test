snf/pa-1.0.0/test.js
200
