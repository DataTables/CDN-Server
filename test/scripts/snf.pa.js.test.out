extensionsURL >snf/pa-1.0.0<
extensionsList > ProductA 1.0.0<
source >{source}<
testString >{testString}<
styling >{styling}<
./ProductA-1.0.0/js/productA.styleNoFiles.js


./ProductA-1.0.0/js/productA.js


